docs/README.md

```diff
@@ -5,8 +5,11 @@
 ```
 composer require dereuromark/cakephp-queue
 ```
-
-Enable the plugin within your config/bootstrap.php (unless you use loadAll):
+Load the plugin in your `src/Application.php`'s bootstrap() using:
+```php
+$this->addPlugin('Queue');
+```
+Prior to 3.6.0 enable the plugin within your config/bootstrap.php (unless you use loadAll):
```

Comment:
That is not needed IMO as we already are CakePHP 3.7+.

Reply:
So then just change it to $this->addPlugin('Queue'); since Plugin::load('Queue'); is deprecated?